From 52b5e845e34cb893c954f2faba6d9c8c344821be Mon Sep 17 00:00:00 2001
From: qianmenx <qianx.meng@intel.com>
Date: Thu, 16 May 2019 11:22:03 +0800
Subject: [PATCH 1155/1214] media: intel-ipu4: fix frame counter(x3a) can't
 work issue

fix frame counter(x3a) can't work issue

Change-Id: I7c5b694a1acc44248959be2ed8b16aa390885ab2
Tracked-On: PKT-2588
Tracked-On: #JIIAP-791
Signed-off-by: qianmenx <qianx.meng@intel.com>
Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 drivers/media/i2c/crlmodule/crl_ox03a10_common.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/media/i2c/crlmodule/crl_ox03a10_common.h b/drivers/media/i2c/crlmodule/crl_ox03a10_common.h
index dc7ab02..1765cb2 100644
--- a/drivers/media/i2c/crlmodule/crl_ox03a10_common.h
+++ b/drivers/media/i2c/crlmodule/crl_ox03a10_common.h
@@ -529,6 +529,7 @@ static struct crl_register_write_rep ox03a10_1920_1088_12DCG[] = {
 	{ 0x384d, CRL_REG_LEN_08BIT, 0x14 },
 	{ 0x460a, CRL_REG_LEN_08BIT, 0x0e },
 	/* embedded data */
+	{ 0x484c, CRL_REG_LEN_08BIT, 0x02 },
 	{ 0x3208, CRL_REG_LEN_08BIT, 0x04 },
 	{ 0x483e, CRL_REG_LEN_08BIT, 0x02 },/* frame counter */
 	{ 0x3501, CRL_REG_LEN_08BIT, 0x02 },/* DCG exp */
-- 
2.7.4

